Examples/demo-chat-app/demo-chat-app/demo_chat_appApp.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+import momento
+
+class Momento: ObservableObject {
+    public var topicClient: TopicClientProtocol
+    private let momentoApiKey: String = ""
```

Comment:
User will need to provide their Momento API key here in order to run the app. Can switch to using a token vending machine in the future when we implement `generateDisposableToken`